### PR TITLE
Fix Rails 6.1 tests on master

### PR DESCRIPTION
**Pull request by thimo**
This PR fixes the issues with testing against Rails 6.1. The changes applied here are the same as those already applied on the `bootstrap-5` branch. Also solves a few Rubocop offences.

### CODE REVIEW

**Comment by thimo** [comment]
Github Action "Lint" fails with the following message:

> Could not set up API to Code Review site for Danger
> For your GitHub repo, you need to expose: DANGER_GITHUB_API_TOKEN
> You may also need: DANGER_GITHUB_HOST, DANGER_GITHUB_API_BASE_URL, DANGER_OCTOKIT_VERIFY_SSL

Is there someone who can set this up correctly? Lint only runs on PR's, so it's not a problem for direct commits. Not sure if this is on purpose?

**Comment by lcreid** [comment]
I can take a look, but I didn't set up Danger so it may take a while for me to figure it out.

**Comment by thimo** [comment]
As Danger runs just fine in the Travis tests I can image there being a `DANGER_GITHUB_API_TOKEN` env variable in Travis that you can copy to the Github bootstrap_form project. But I'm neither familiar with Travis nor Danger. Do you have access to Travis?

**Comment by thimo** [comment]
Additionally removed `.rubocop_todo.yml` and mostly applied the same Rubocop offences fixes as I did to the `bootstrap-5` branche.

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,81 +0,0 @@
-# This configuration was generated by
```
Comment (lcreid) [inline .rubocop_todo.yml:1]: Nice! It's great to see this file disappear.

**Review by lcreid — APPROVED** [review]
Looks good. Thanks so much!

**Pull request merged**